static/app/components/events/aiAutofix/useAiAutofix.tsx

```diff
@@ -0,0 +1,53 @@
+import {useCallback, useState} from 'react';
+
+import {AutofixData, GroupWithAutofix} from 'sentry/components/events/aiAutofix/types';
+import {useApiQuery} from 'sentry/utils/queryClient';
+import useApi from 'sentry/utils/useApi';
+
+const POLL_INTERVAL = 5000;
+
+export const useAiAutofix = (group: GroupWithAutofix) => {
+  const api = useApi();
+  const {
+    data: apiAutofixData,
+    isError,
+    refetch: dataRefetch,
+    error,
+  } = useApiQuery<AutofixData | null>([`/issues/${group.id}/ai-autofix/`], {
+    staleTime: Infinity,
+    retry: false,
+    enabled: !group.metadata?.autofix, // Enabled only when no autofix data present
```

Comment:
Does this work when the group doesn't have an autofix to start? It seems like `triggerAutofix` below would do a single refetch, but if this would stay `enabled: false`

Reply:
Yep it does, and it also works when there is already an autofix and trigger is pressed again